lang/java/BUILD.md

```diff
@@ -44,6 +46,12 @@ To perform "fast" build, which skips tests and all code quality enforcement tool
 
     mvn clean install -DskipTests -TC1 -P!code-quality
 
+
+  * Please follow the following wiki for all running all java builds [Java Build Wiki](https://cwiki.apache.org/confluence/display/REEF/Linux), the concept of ulimit increase allows more file descriptors to be opened for some of the unit tests in REEF
```

Comment:
I would consider the following.

The REEF unit tests require a number of open files which is greater then the default open file limit on a number of Linux distributions such as Ubuntu 16.04 / 16.10. This limit is controlled in the shell by the "ulimit -n" command. Please see the follow documention on how adjust the ulimit........